src/request/repository/converter/request-converter.js

```diff
@@ -139,19 +140,25 @@ var getStatusCodeText = (function() {
 })();
 var setupIndex = function(request, type, payload) {
     if (type == 'begin-request') {
-        request._requestStartTime = payload.requestStartTime;
-        request._requestMethod = payload.requestMethod;
-        request._requestUrl = (payload.requestPath || '') + (payload.requestQueryString || '');
+        var url = parse(payload.url);
+        payload.path = url.pathname;
+        payload.query = url.query;
+        
+        request._requestStartTime = payload.startTime;
+        request._requestMethod = payload.method;
+        request._requestUrl = payload.path + payload.query;
         request._requestIsAjax = payload.requestIsAjax;
     }
     if (type == 'end-request') {
-        request._responseStatusCode = payload.responseStatusCode; 
-        request._responseStatusText = getStatusCodeText(payload.responseStatusCode);
-        request._responseContentType = payload.responseContentType;
-        request._responseContentCategory = getContentTypeCategory(payload.responseContentType);  
+        var contentType = payload.headers && payload.headers['Content-Type'];
```

Comment:
You bring up an interesting point. Slight pain here... in theory there are a bunch of different ways you could capitalize the titles/properties. Unfortunately there isn't any nice way that I know of to access/index a property in a case insensitive way. We could loop over each property in the object and then test a case insensitive match, but that feels very ugly. This might mean its worth promoting `ContentType` into its own first level property. @nikmd23